include/felspar/memory/holding_pen.hpp

```diff
@@ -54,6 +54,16 @@ namespace felspar::memory {
             return *this;
         }
 
+        bool operator==(const holding_pen &hp) const {
```

Comment:
east const

Reply:
The better way of doing these is as a friend:
```
friend bool operator==(holding_pen const &l, holding_pen const &r);
```
There are some advantages in build speed and it makes the operator more symmetrical